frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -1,5 +1,186 @@
+        <TextInput dataKey="person.addressCity" label="City" />
+        <TextInput dataKey="person.addressState" label="State" />
+        <TextInput dataKey="person.addressZip" label="Zip Code" />
+        <TextInput dataKey="person.phone" label="Phone" />
+        <TextInput dataKey="person.email" label="Email" />
+        <small className="web-form-input">
+          Check your email. You will receive information and documents at this
+          email address.
+        </small>
+      </Section>
+
+      <Section name="Case Information">
+        <Radio
+          dataKey="case.courtType"
+          label="Court Type"
+          options={courtTypeOptions}
+        />
+
+        {data.case.courtType === "District" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="District Courts"
+            groupOptions={DistrictCourtList}
+          />
+        )}
+        {data.case.courtType === "Justice" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label="Case Number" />
+        <TextInput dataKey="case.judgeName" label="Judge Full Name" />
+        <TextInput
+          dataKey="case.commissionerName"
+          label="Commissioner (Domestic Cases)"
+        />
+      </Section>
+
+      <Section name="1. Documents Served">
+        <small className="web-form-input">
+          The following documents were served by the method described below
+        </small>
+        <small className="web-form-input">(Choose all that apply:)</small>
+        <Checkbox
+          dataKey="proofOfCompletedService_case.summons"
+          label="Summons (File or attach copy)"
+        />
+        <Checkbox
+          dataKey="proofOfCompletedService_case.complaintOrPetition"
+          label="Complaint or Petition"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.amendedComplaintOrPetition"
+          label="Amended Complaint or Petition"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.noticeOfDivorceEducationRequirements"
+          label="Notice of Divorce Education Requirements"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.noticeOfURCP26.1"
+          label="Notice of URCP 26.1 Disclosure and Discovery Requirements in Domestic Relations Actions"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.noticeOfURCP26.3"
+          label="Notice of URCP 26.3 Disclosure Requirements in Unlawful Detainer Actions"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.parentingPlan"
+          label="Parenting Plan"
+        />
+
+        <Checkbox
+          dataKey="proofOfCompletedService_case.other"
+          label="Other (describe)"
+        />
+        {/* {case.proofOfCompletedService_case && ( 
```

Comment:
Do we need this? Or can it be removed?

Reply:
Removed and now describe field appears only if Other is checked